ovs/lib/mdsservice.py

```diff
@@ -672,8 +677,10 @@ def _add_suitable_nodes(local_importance, local_safety, services_to_recycle=list
         # Master must be local
         # Master cannot be overloaded
         # Master must be in primary domain (if no domains available, this check is irrelevant because all StorageRouters will match)
+        timeout = 300
```

Comment:
Let's introduce an option to overrule this? Timeout section in config management? Would be cool for documentation!